checks.d/wmi_check.py

```diff
@@ -42,7 +42,7 @@ def check(self, instance):
 
         wmi_sampler = self._get_wmi_sampler(
             instance_key,
-            wmi_class, properties,
+            wmi_class, properties, tag_by,
```

Comment:
Nitpick: I prefer naming the parameter explicitely when it's an optional parameter, so `tag_by=tag_by` here :)